### PR TITLE
Editorial fixes: Address Barry's and Robert's comments

### PR DESCRIPTION
**Pull request by renghardt**
This PR has fixes for Barry's and Robers's comments, which are all editorial.

However, please double-check and state an opinion on the definition of a Transport Services system and on the attempt to provide more context to make the document easier to understand for readers not yet familiar with TAPS.

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -122,15 +122,16 @@ Services and features provided by transport protocols have been cataloged in {{?
...
 applications.  It examines Transport Layer Security (TLS), Datagram Transport Layer Security (DTLS),
-IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), SRTP (with DTLS),
+IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), Secure Real-time Transport Protocol (SRTP) with DTLS,
 WireGuard, CurveCP, and MinimaLT. For each protocol, this document provides a brief description.
 Then, it describes the interfaces between these protocols and transports in {{transport-interface}}
 and the interfaces between these protocols and applications in {{application-interface}}.
 
-Selected protocols represent a superset of functionality and features a Transport Services system may
+A Transport Services system exposes an interface to different Transport Services to applications. Further, a Transport Services system maps the Transport Service that an application has requested to a specific protocol stack, including transport protocols and security protocols.
+The security protocols included in survey represent a superset of functionality and features a Transport Services system may
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md]:
```suggestion
The security protocols included in this survey represent a superset of functionality and features a Transport Services system may
```

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -356,7 +359,7 @@ OpenVPN {{OpenVPN}} is a commonly used protocol designed as an alternative to
 IPsec. A major goal of this protocol is to provide a VPN that is simple to
 configure and works over a variety of transports. OpenVPN encapsulates either
 IP packets or Ethernet frames within a secure tunnel and can run over either UDP or TCP.
-For key establishment, OpenVPN can use TLS as a handshake protocol or pre-shared keys.
+For key establishment, OpenVPN can use TLS as a handshake protocol or it can use pre-shared keys.
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md]:
```suggestion
For key establishment, OpenVPN can either use TLS as a handshake protocol or use pre-shared keys.
```

File: draft-ietf-taps-transport-security.md [hunk 3]
```diff
@@ -122,15 +122,16 @@ Services and features provided by transport protocols have been cataloged in {{?
...
 and identifying the interfaces between these protocols and both transport protocols and
 applications.  It examines Transport Layer Security (TLS), Datagram Transport Layer Security (DTLS),
-IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), SRTP (with DTLS),
+IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), Secure Real-time Transport Protocol (SRTP) with DTLS,
 WireGuard, CurveCP, and MinimaLT. For each protocol, this document provides a brief description.
 Then, it describes the interfaces between these protocols and transports in {{transport-interface}}
 and the interfaces between these protocols and applications in {{application-interface}}.
 
-Selected protocols represent a superset of functionality and features a Transport Services system may
+A Transport Services system exposes an interface to different Transport Services to applications. Further, a Transport Services system maps the Transport Service that an application has requested to a specific protocol stack, including transport protocols and security protocols.
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md]:
```suggestion
A Transport Services system exposes an interface for applications to access various transport protocol features, including transports that offer security.
```

**Review by tfpauly — CHANGES_REQUESTED** [review]

File: draft-ietf-taps-transport-security.md (hunk 3 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: How about the following?

"A Transport Services system exposes an application interface for accessing various (secure) transport protocol features."

**Review by chris-wood — APPROVED** [review]
Approved pending the suggested simplification!

File: draft-ietf-taps-transport-security.md (hunk 3 above)
Comment (renghardt) [reply draft-ietf-taps-transport-security.md]: Thanks for the suggestions! I like both and I'm going with the latter.

**Review by chris-wood — APPROVED** [review]

**Pull request merged**